### ISSUE TITLE
Possible bug during login

### ISSUE THREAD

**Issue by stefan-niedermann** [stefan-niedermann/nextcloud-notes#702]
@MoritzGiessmann mentioned a possible bug while logging in. He talked about a "not understandable error message during entering the URL".This issue might already be solved with the new SSO.

@MoritzGiessmann might you have the time to comment what exactly you saw and if this issue still occurs with the current version (2.5.0)?

Thank you im advance

**Comment by MoritzGiessmann** [comment]
Hi!
I "returned" the app when the bug was occuring, so I can only explain the bug from my memory.
If I remember it correctly when starting the app I saw a not so self explanatory input field. I put in my Nextcloud URL and after that the screen showed lots of strange error text I couldn’t figure out what to change or what I was doing wrong. There also wasn’t a way back to correct the issue.
The normal Nextcloud app now also has a markdown editor, so I didn’t bother reinstalling Nextcloud-Notes.
I hope that helps.

**Comment by stefan-niedermann** [comment]
Thank you for your response.

Would you mind to give it another try? This might be incredibly helpful for other users.

If you don't want to purchase it again at the Play Store, i can send you the (signed) app via E-Mail or you can grab it for free on [F-Droid](https://f-droid.org/repo/it.niedermann.owncloud.notes_71.apk).

> The normal Nextcloud app now also has a markdown editor

As far is i know this is only a viewer without editing capabilities? :eyes:

**Comment by MoritzGiessmann** [comment]
Hi! I installed it from F-Droid and it even recognized my already set up Nextcloud Account from the Android account. I also tried using it just using the URL and it worked just fine. 
So the problem appears to be gone.
Thanks for reaching out!

Here’s a screenshot of what the official Nextcloud app does, when I select a .txt or .md file:
![nextcloud-app-md-editor](https://user-images.githubusercontent.com/184818/73125598-c75c6c00-3fa8-11ea-874f-cd816cb412c1.jpeg)
